### PR TITLE
[PAR-WDFT]-PASCAL-BESSON [ongoing]

### PR DESCRIPTION
**Pull request by pascalbes**

### CODE REVIEW

File: starter-code/js/queue.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    queueInitial=getQueue();
+
+    let queueFinal = queueInitial.dequeue(document.getElementById("queue-input").value);
+
+    if (queueFinal === 'Queue Underflow') {
+        alert(queueFinal)
+    }
+    else {
+        queueInitial.queueControl.pop();
+        displayQueue(queueInitial);
```
Comment (frankmarmier) [inline starter-code/js/queue.js]: Here you are using the pop method on the queueControl property, you have created methods to interact with your queue

**Comment by frankmarmier** [comment]
I noticed that for both of your queue and stack classes, in the DOM, you are interacting directly on the array properties that are within the classes instead of using the methods you've created.

I'll let you fix it :)

**Comment by pascalbes** [comment]
v2 online ;)

**Comment by frankmarmier** [comment]
Closing the PR I forgot to close !

**Pull request closed without merging**